### DISCUSSION TITLE
405 Method Not Allowed on web requests from browser

### DISCUSSION THREAD

**Discussion by sveltarne** [pocketbase/pocketbase#992] [Q&A] [answered]
I've attempted 2 web requests - one to create a user and one to log in via email, and both of them succeed using Insomnia, a REST client, but return a 503 error from the browser.

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure how to reproduce it. Please provide a little more info how you are sending the requests (eg. what content-type you are using, the HTTP method, etc.).

**Reply by sveltarne** [reply]
I'm sending a POST request and the content type is `application/json`. The browser is firefox, the operating system is Windows 11, and I am sending web requests directly, not using the SDK. Also, the website is on sveltekit and using fetch for the requests.

**Reply by sveltarne** [reply]
The same issue also happened on ubuntu. The rest client worked fine, but browser returned 405 Method Not Allowed

**Reply by ganigeorgiev** [reply]
If you are able to make the request with the rest client, then probably something is wrong with the `fetch` call. Are you sure that the url is correct? Could you post a code sample with the request options you are sending?

**Reply by sveltarne** [reply]
Here's the file for all my web requests: https://github.com/sveltarne/observer/blob/master/src/lib/scripts/req.ts

The two functions that are interacting with pocketbase are createUser and loginEmail.

The URL is the exact same as the rest client, so I doubt that's the problem, but I made sure to try it with 127.0.0.1 instead of localhost and same error.

**Reply by ganigeorgiev** [reply]
The concatenated url is not correct (you are missing the `/api/` part). The login url should be:
```
// PocketBase v0.7.x
http://127.0.0.1:8090/api/users/auth-via-email
{ email: "yourEmail", password: "123456", passwordConfirm: "123456"}

// PocketBase v0.8.0-rc
http://127.0.0.1:8090/api/collections/users/auth-with-password
{ identity: "usernameOrEmail", password: "123456", passwordConfirm: "123456"}
```
